card/mod/account/spec/set/all/account_spec.rb

```diff
@@ -1,140 +1,133 @@
 # -*- encoding : utf-8 -*-
 
-describe Card::Set::All::Account do
+RSpec.describe Card::Set::All::Account do
   describe "accountable?" do
     it "is false for cards with *accountable rule off" do
       expect(Card["A"].accountable?).to eq(false)
     end
 
-    it "is true for cards with *accountable rule on" do
-      Card::Auth.as_bot do
-        Card.create name: "A+*self+*accountable", content: "1"
-        Card.create name: "*account+*right+*create",
-                    content: "[[Anyone Signed In]]"
-      end
+    it "is true for cards with *accountable rule on", as_bot: true do
```

Comment:
new test api to get rid of all these `Card::Auth.as_bot` block in our test

Reply:
nice!